Code/src/main/java/nl/utwente/group10/ui/Main.java

```diff
@@ -56,11 +62,13 @@ public void start(Stage stage) throws Exception {
                     "available. Executing programs will not be enabled. We " +
                     "strongly recommend you install GHC, for example by " +
                     "installing the Haskell Platform (haskell.org/platform).";
-            tactilePane.getChildren().add(new CustomAlert(tactilePane, msg));
+            CustomAlert alert = new CustomAlert(tactilePane, msg);
+            tactilePane.getChildren().add(alert);
+            alert.relocate(tactilePane.getWidth() / 2 - 200, tactilePane.getHeight() / 2 - 200);
```

Comment:
Correctly positioning the CustomAlert could be a PR. (Combine with whitespace changes in CustomAlert itself.)